examples/blog/invoiceGenerator/src/interfaces/index.d.ts

```diff
@@ -6,7 +6,7 @@ export interface ICompany {
     city: string;
     email: string;
     website: string;
-    logo: any;
+    logo: any; // eslint-disable-line
```

Comment:
```suggestion
    logo?: null | { url: string };
```

Added types based on the usage 🚀

Reply:
done